fedimint-ui/apps/guardian-ui/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "guardian-ui",
+  "name": "@fedimint/guardian-ui",
   "version": "0.1.0",
   "private": true,
   "scripts": {
```

Comment:
the `dev` script should define an ENV VAR for port. 
`"dev": "PORT=3001 react-scripts start"`

Otherwise, it conflicts with gateway-ui when running `yarn dev` from `fedimint-ui`.
<img width="791" alt="Screenshot 2023-05-10 at 9 26 38 PM" src="https://github.com/fedimint/fedimint/assets/39040273/de763f4e-5bdb-4658-be5e-7862644338a9">

Reply:
I've removed `dev` script from `fedimint-ui` workspace root for the following reasons:
If a dev wan'ts to contribute to both apps, they would have to cd to app and run the proper start command, which should prompt the correct port selection and avoid conflicts